train.py

```diff
@@ -332,14 +346,18 @@
 
 
 model_modes = ["Random Forest", "Regression", "CatBoostClassifier"]
-full_eval = WorldCerealEval(train_df, val_df, spatial_inference_savedir=model_logging_dir)
+full_eval = WorldCerealEval(
+    train_df, val_df, spatial_inference_savedir=model_logging_dir, dekadal=dekadal
+)
 results, finetuned_model = full_eval.finetuning_results(model, sklearn_model_modes=model_modes)
 logger.info(json.dumps(results, indent=2))
-if finetuned_model is not None:
-    model_path = model_logging_dir / Path("models")
-    model_path.mkdir(exist_ok=True, parents=True)
-    finetuned_model_path = model_path / "finetuned_model.pt"
-    torch.save(model.state_dict(), finetuned_model_path)
+
+#if finetuned_model is not None:
+model_path = model_logging_dir / Path("models")
+model_path.mkdir(exist_ok=True, parents=True)
+finetuned_model_path = model_path / "finetuned_model.pt"
```

Comment:
https://github.com/WorldCereal/presto-worldcereal/pull/35/files#r1547829997 

but here `finetuned_model.pt` is loaded

Reply:
that's where the `finetuned_model.pt` is saved after finetuning the `dekadal_model.pt` (which is being loaded here https://github.com/WorldCereal/presto-worldcereal/blob/dataset-10days/train.py#L340 )